operations/iuf/workflows/configuration.md

```diff
@@ -203,11 +203,18 @@ required for initial installation scenarios.
   - Configure SAT authentication via `sat auth`
   - Generate SAT S3 credentials
   - Configure system revision information via `sat setrev`
-- UAS
-  - Configure UAS network settings 
-    - The network settings for UAS must match the WLM to allow job submission from UAIs
-- Badger
-  - Update CSM Diags network attachment definition
+- SLURM
+  - UAS
+    - Configure UAS network settings 
+      - The network settings for UAS must match the SLURM WLM to allow job submission from UAIs
+  - Badger
```

Comment:
```suggestion
  - CSM Diags
```

Reply:
The jira I opened on this I inadvertently used the name "Badger" rather than "CSM Diags".   The WLM docs refer to this change as being for "CSM Diags", so that is better terminology here.